### ISSUE TITLE
Fix a11y issues on the Data Tool

### ISSUE THREAD

**Issue by shawnbot** [RTICWDT/college-scorecard#1475]
I neglected to add `/data/` to the list of URLs that gets tested for accessibility. The failures are captured in [this build](https://circleci.com/gh/18F/college-choice/269), and should be easy to fix.

**Comment by brownpl** [comment]
Working on a 508 branch and I've incorporated the data section into the a11y tests.
